macos/Sources/Features/QuickTerminal/QuickTerminalSize.swift

```diff
@@ -33,6 +33,7 @@ struct QuickTerminalSize {
             case GHOSTTY_QUICK_TERMINAL_SIZE_PIXELS:
                 self = .pixels(cStruct.value.pixels)
             default:
+                assertionFailure()
```

Comment:
without the fix, this gets hit quite often in debug builds for me (with config `quick-terminal-size = 75%, 50%` on M1 Pro running macOS 26.1).

figured it's worthwhile leaving this here (I saw many similar ones in `macos/Sources/Ghostty/Ghostty.App.swift`), but it isn't strictly needed and can be removed